HADES.Core/src/EnhancedHealth.cs

```diff
@@ -11,14 +12,13 @@ public class EnhancedHealth : HADESEnhancement
         public float HealthPercentage { get; private set; }
 
         private float CurrentHealth => GM.GetPlayerHealth();
-        private float RegenCap => HADESConfig.EnhancedHealth.RegenCap;
-        private float RegenDelay => HADESConfig.EnhancedHealth.RegenDelay;
         private float RegenSpeed => HADESConfig.EnhancedHealth.RegenSpeed;
 
-        private GameObject HealthBars => Player.HealthBar;
-        
-        private float _initialHealth;
-        
+        private float RegenToGo;
+        private float currentRegenDelayLength;
+        private float healthMonitor;
```

Comment:
Use `_hungarianNotation` for private fields, private properties should use `CamelCase`